bugsnag-cli-wrapper.js

```diff
@@ -0,0 +1,151 @@
+const { exec } = require('child_process');
+
+/**
+ *
+ * @typedef {Object} BugsnagUploadOptions
+ * @property {?string} apiKey - The API key for authentication. Optional.
```

Comment:
is apiKey actually optional?

Reply:
Yes as it will try and get them from the project if it's not provided. All of them are optional